docs/python_docs/python/tutorials/extend/customop.md

```diff
@@ -0,0 +1,218 @@
+# Creating custom operators with numpy
```

Comment:
@ptrendx pointed out that AMP docs are missing

Reply:
that's correct, some tutorials are missing and will be part of a post-launch re-alignment